templates/firefox-ios.xliff

```diff
@@ -4125,14 +4156,34 @@ Private Tab</source>
         <source>Secure connection</source>
         <note>Accessibility label for the security icon in url bar</note>
       </trans-unit>
+      <trans-unit id="TabLocation.LockButton.AccessibilityLabel.v122" xml:space="preserve">
+        <source>Tracking Protection</source>
+        <note>Accessibility label for the lock / tracking protection button on the URL bar</note>
+      </trans-unit>
+      <trans-unit id="TabLocation.LockButton.LargeContentTitle.v122" xml:space="preserve">
+        <source>Tracking Protection</source>
+        <note>Large content title for the lock button. This title is displayed when using accessible font sizes is enabled</note>
+      </trans-unit>
       <trans-unit id="TabLocation.Share.A11y.Label.v119" xml:space="preserve">
         <source>Share this page</source>
         <note>Accessibility label for the share button in url bar</note>
       </trans-unit>
+      <trans-unit id="TabLocation.ShareButton.AccessibilityLabel.v122" xml:space="preserve">
+        <source>Share</source>
+        <note>Large content title for the share button. This title is displayed when using accessible font sizes is enabled</note>
+      </trans-unit>
       <trans-unit id="TabLocation.Shopping.A11y.Label.v120" xml:space="preserve">
         <source>Review Checker</source>
         <note>Accessibility label for the shopping button in url bar</note>
       </trans-unit>
+      <trans-unit id="TabsButton.Accessibility.LargeContentTitle.v122" xml:space="preserve">
+        <source>Show Tabs, %@</source>
```

Comment:
@adudenamedruby Not sure since this is accessibility: shouldn't it be a colon instead of a coma here?